Manito/Manito/Screens/CreateRoom/CreateRoomViewController.swift

```diff
@@ -7,23 +7,118 @@
 
 import UIKit
 
-class CreateRoomViewController: UIViewController {
+import SnapKit
 
+class CreateRoomViewController: BaseViewController {
+
+    // MARK: - Property
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "방 생성하기"
+        label.font = .font(.regular, ofSize: 34)
+        return label
+    }()
+    
+    lazy var closeButton: UIButton = {
+        let btn = UIButton(type: .system)
+        btn.setImage(UIImage(systemName: "xmark.circle.fill"), for: .normal)
+        btn.tintColor = .lightGray
+        btn.addTarget(self, action: #selector(didTapCloseButton), for: .touchUpInside)
+        return btn
+    }()
+    
+    lazy var nextButton: UIButton = {
+        let btn = UIButton(type: .system)
+        btn.setTitle("다음", for: .normal)
+        btn.titleLabel?.font = .font(.regular, ofSize: 20)
+        btn.tintColor = .white
+        btn.backgroundColor = UIColor.mainRed
+        btn.layer.cornerRadius = 30
+        btn.addTarget(self, action: #selector(didTapnextButoon), for: .touchUpInside)
+        return btn
+    }()
+    
+    private let nameView: InputNameView = {
+        let view = InputNameView()
+        return view
+    }()
+    
+    private let personView: InputPersonView = {
+        let view = InputPersonView()
+        return view
+    }()
+    
+    private let dateView: InputDateView = {
+        let view = InputDateView()
+        return view
+    }()
+    
+    // MARK: - Life Cycle
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        
     }
     
+    override func render() {
 
-    /*
-    // MARK: - Navigation
+        view.addSubview(titleLabel)
+        titleLabel.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide).inset(66)
+            $0.leading.equalTo(view).inset(16)
```

Comment:
p1;

```suggestion
            $0.leading.equalTo(view.safeAreaLayoutGuide).inset(16)
```
요것이 빠진 거 같습니다.!

Reply:
수정완료 [4e5e711](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/15/commits/4e5e7112b41af229f1c76bba9cd9fdb09662cbce)